tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/model/PersonalInfo.java

```diff
@@ -60,6 +60,7 @@
   private Calendar dateOfGraduation;
 
   @Parameter
+  @Optional
```

Comment:
add a new parametr instead

Reply:
if i add a new parameter intead of this, this would still be required and wouldmn't fit the scenario it tries to reproduce